build.gradle

```diff
@@ -22,7 +22,7 @@ distTar {
     archiveName "shuttle.tgz"
 }
 
-mainClassName = "com.openlattice.shuttle.ShuttleServer"
+mainClassName = "com.openlattice.shuttle.ShuttleKt"
```

Comment:
I don't see `ShuttleKt` anywhere.

Reply:
It's a Kotlin thing. When Shuttle.kt contains package level functions they are placed inside of a class called ShuttleKt.